exdbproject/settings.py

```diff
@@ -134,6 +138,11 @@
 JS_FILE_EXCLUDED_DIRS = ['coverage', 'instrumented_static', 'libraries', 'htmlcov']
 PY_FILE_EXCLUDED_DIRS = ['migrations']
 
+# restricted access middleware permissions
```

Comment:
There should be some more documentation for this setting. For instance: what arguments are supplied to the function and what it is meant to return.